src/chains/index.ts

```diff
@@ -0,0 +1,232 @@
+import { Chain } from 'viem/chains';
+
+import backendNetworks from '../references/networks.json';
+
+import { ChainId, BackendNetwork, BackendNetworkServices, chainHardhat, chainHardhatOptimism } from './types';
+import { transformBackendNetworksToChains } from './utils/backendNetworks';
+import { gasUtils } from '@/utils';
+import { useConnectedToHardhatStore } from '@/state/connectedToHardhat';
+
+const IS_TESTING = process.env.IS_TESTING === 'true';
```

Comment:
curious why this instead of `import { IS_TESTING } from 'react-native-dotenv';` that is used mostly everywhere else